extensions/wereadsync/src/services/autoSync.ts

```diff
@@ -0,0 +1,145 @@
+import { getLocalStorageItem, setLocalStorageItem, showToast, Toast, getPreferenceValues } from "@raycast/api";
+import { showFailureToast } from "@raycast/utils";
+import { SyncService } from "./syncService";
+
+interface Preferences {
+  wereadCookie: string;
+  readwiseToken: string;
+  autoSyncEnabled: boolean;
+  autoSyncInterval: string;
+}
```

Comment:
This is already a global type in the `raycast-env.d.ts`. Feel free to drop it gracefully.